copycat/api/src/main/java/org/apache/kafka/copycat/sink/SinkTaskContext.java

```diff
@@ -59,6 +60,22 @@ public void offset(Map<TopicPartition, Long> offsets) {
     }
 
     /**
+     * Get the topic partition that throws Transient exception during message delivery
+     * @return the topic partition that has error
+     */
+    public TopicPartition erorrTopicPartition() {
+        return errorTopicPart;
+    }
+
+    /**
+     * Set the error topic partition
+     * @param errorTopicPart the topic partition with error
+     */
+    public void errorTopicPartition(TopicPartition errorTopicPart) {
```

Comment:
I assume this is the method that is supposed to be used by connectors. Can the comment explain in more detail how it is supposed to work? What if you have multiple TopicPartitions with errors?